### ISSUE TITLE
React Query: Extra  semicolon ; generating invalid TypeScript

### ISSUE THREAD

**Issue by melloware** [anymaniax/orval#521]
## What versions are you using?

**Operating System:** ALL
**Package Version:** 6.9.2
**Browser Version:** ALL

**OpenAPI YAML:**
```yaml
---
openapi: 3.0.3
info:
  title: Test Orval Bug
  description: Test Orval Bug
  version: 1.0.0
paths:
  /entity/cars:
    get:
      tags:
      - Car Entity Resource
      parameters:
      - name: request
        in: query
        schema:
          type: string
      responses:
        "200":
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/QueryResponseCarEntity'
    post:
      tags:
      - Car Entity Resource
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/CarEntity'
      responses:
        "200":
          description: OK
    delete:
      tags:
      - Car Entity Resource
      responses:
        "200":
          description: OK
  /entity/cars/{id}:
    get:
      tags:
      - Car Entity Resource
      parameters:
      - name: id
        in: path
        required: true
        schema:
          format: int64
          minimum: 0
          type: integer
      responses:
        "200":
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/CarEntity'
    put:
      tags:
      - Car Entity Resource
      parameters:
      - name: id
        in: path
        required: true
        schema:
          format: int64
          minimum: 0
          type: integer
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/CarEntity'
      responses:
        "200":
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/CarEntity'
    delete:
      tags:
      - Car Entity Resource
      parameters:
      - name: id
        in: path
        required: true
        schema:
          format: int64
          minimum: 0
          type: integer
      responses:
        "200":
          description: OK
components:
  schemas:
    CarEntity:
      required:
      - vin
      - make
      - model
      - color
      type: object
      properties:
        id:
          format: int64
          type: integer
        vin:
          pattern: \S
          type: string
        make:
          pattern: \S
          type: string
        model:
          pattern: \S
          type: string
        year:
          format: int32
          minimum: 1970
          type: integer
        color:
          pattern: \S
          type: string
        price:
          minimum: 0
          type: number
    QueryResponseCarEntity:
      type: object
      properties:
        totalRecords:
          format: int64
          type: integer
        records:
          type: array
          items:
            $ref: '#/components/schemas/CarEntity'

```

## What are the steps to reproduce this issue?

When generating this method for `delete` which has no parameters it generates an extra ` semicolon` after `TVariables` which make its invalid TypeScript.

![image](https://user-images.githubusercontent.com/4399574/182409638-32822092-dfd3-429a-9613-95f9dc8fb823.png)

```ts
export const useDeleteEntityCars = <TError = ErrorType<unknown>,
    TVariables = void,
    TContext = unknown>(options?: { mutation?: UseMutationOptions<Awaited<ReturnType<ReturnType<typeof useDeleteEntityCarsHook>>>, TError, TVariables, TContext>, }
    ) => {
    const { mutation: mutationOptions } = options ?? {}

    const deleteEntityCars = useDeleteEntityCarsHook()


    const mutationFn: MutationFunction<Awaited<ReturnType<ReturnType<typeof useDeleteEntityCarsHook>>>, TVariables> = () => {
        ;

        return deleteEntityCars()
    }

    return useMutation<Awaited<ReturnType<typeof deleteEntityCars>>, TError, TVariables, TContext>(mutationFn, mutationOptions)
}
```

There is no parameter for the delete but in the Mutation its generating the blank comma.

![image](https://user-images.githubusercontent.com/4399574/182409250-a715cfb1-8b2e-4c75-ae9d-66cf59e4266b.png)

**Comment by anymaniax** [comment]
Hello @melloware, did a fix will release that asap

**Comment by melloware** [comment]
As always you rock!

**Comment by anymaniax** [comment]
Fixed with `6.9.3` normally

**Comment by melloware** [comment]
works like a charm!
